src/main/java/ch/njol/skript/structures/StructCommand.java

```diff
@@ -83,7 +83,7 @@ public class StructCommand extends Structure {
 	public static final Priority PRIORITY = new Priority(500);
 
 	private static final Pattern
-		COMMAND_PATTERN = Pattern.compile("(?i)^command /?(\\S+)\\s*(\\s+(.+))?$"),
+		COMMAND_PATTERN = Pattern.compile("(?i)^command\\s+/?(\\S+)\\s*(\\s+(.+))?$"),
```

Comment:
I am concerned that these patterns accept so much. You are asking for any whitespace here (including weird low-index unicode characters, 0-width spaces, maybe some control characters, etc.) and the non-whitespace group will match characters that are illegal in a command namespace. It might be better to be a bit more selective here.

Reply:
I could've used ` +` but anywhite `\\s` space is also used in the rest of the regex, do you think we should change all that? (tho that could be a breaking change)

Reply:
It should just be one, not unlimited.

Reply:
That's the original issue #5414
Skript does allow multiple whitespaces in many places, therefore this better follow the system

Reply:
Ya but like TP said, we shouldn't assume the marer passes, just make the matcher false so that it errors properly to the user instead of an exception.

Reply:
Yea that will be added in a bit, but the multiple whitespaces support is still a thing to be fixed